CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+
+To get a full outline of the changes, we invite you to review the official changelog below. This release does include a Breaking Change to existing v0.10.x databases. If you need help migrating an existing deployment, reach out at [hello@source.network](mailto:hello@source.network) or join our Discord at https://discord.gg/w7jYQVJ/.
+
+### Features
+
+* Update corelog to 0.0.7 ([#2547](https://github.com/sourcenetwork/defradb/issues/2547))
+* Move relation field properties onto collection ([#2529](https://github.com/sourcenetwork/defradb/issues/2529))
+* Lens runtime config ([#2497](https://github.com/sourcenetwork/defradb/issues/2497))
+* Add P Counter CRDT ([#2482](https://github.com/sourcenetwork/defradb/issues/2482))
+* Add Access Control Policy ([#2338](https://github.com/sourcenetwork/defradb/issues/2338))
+* Force explicit primary decl. in SDL for one-ones ([#2462](https://github.com/sourcenetwork/defradb/issues/2462))
+* Allow mutation of col sources via PatchCollection ([#2424](https://github.com/sourcenetwork/defradb/issues/2424))
+* Add Defra-Lens support for branching schema ([#2421](https://github.com/sourcenetwork/defradb/issues/2421))
+* Add PatchCollection ([#2402](https://github.com/sourcenetwork/defradb/issues/2402))
+
+### Fixes
+
+* Return correct results from one-many indexed filter ([#2579](https://github.com/sourcenetwork/defradb/issues/2579))
+* Handle compound filters on related indexed fields ([#2575](https://github.com/sourcenetwork/defradb/issues/2575))
+* Add check to filter result for logical ops ([#2573](https://github.com/sourcenetwork/defradb/issues/2573))
+* Make all array kinds nillable ([#2534](https://github.com/sourcenetwork/defradb/issues/2534))
+* Allow update when updating non-indexed field ([#2511](https://github.com/sourcenetwork/defradb/issues/2511))
+
+### Documentation
+
+* Add data definition document ([#2544](https://github.com/sourcenetwork/defradb/issues/2544))
+
+### Refactoring
+
+* Merge collection UpdateWith and DeleteWith ([#2531](https://github.com/sourcenetwork/defradb/issues/2531))
+* DB transactions context ([#2513](https://github.com/sourcenetwork/defradb/issues/2513))
+* Add NormalValue ([#2404](https://github.com/sourcenetwork/defradb/issues/2404))
+* Clean up client/request package ([#2443](https://github.com/sourcenetwork/defradb/issues/2443))
+* Rewrite convertImmutable ([#2445](https://github.com/sourcenetwork/defradb/issues/2445))
+* Unify Field Kind and Schema properties ([#2414](https://github.com/sourcenetwork/defradb/issues/2414))
+* Replace logging package with corelog ([#2406](https://github.com/sourcenetwork/defradb/issues/2406))
+
+### Testing
+
+* Add more ACP integration tests ([#2583](https://github.com/sourcenetwork/defradb/issues/2583))
```

Comment:
thought: not sure if this worth mentioning as we just added ACP

Reply:
Should have been a `(i)` commit. +1 on the thought to remove this from changelog.

Reply:
✅